projects/packages/jetpack-mu-wpcom/src/features/wpcom-block-editor-nux/src/disable-core-nux.js

```diff
@@ -22,16 +18,7 @@ subscribe( () => {
 		dispatch( 'core/nux' ).disableTips();
 		dispatch( 'automattic/wpcom-welcome-guide' ).setShowWelcomeGuide( true );
 	}
-	if ( select( 'core/edit-post' )?.isFeatureActive( 'welcomeGuide' ) ) {
-		dispatch( 'core/edit-post' ).toggleFeature( 'welcomeGuide' );
```

Comment:
Prevent the disabling of the core welcome guide

Reply:
Same as above.